ssr-express/spec/lib/AmpSsrMiddlewareSpec.js

```diff
@@ -19,8 +19,8 @@ const MockExpressResponse = require('mock-express-response');
 const AmpSsrMiddleware = require('../../lib/AmpSsrMiddleware');
 
 class TestTransformer {
-  async transformHtml(body, options) {
-    return 'transformed: ' + options.ampUrl;
+  transformHtml(body, options) {
+    return Promise.resolve('transformed: ' + options.ampUrl);
```

Comment:
Transformers do not need to be async at all.

Reply:
This is actually replacing the DomTransformer, which returns a Promise (https://github.com/ampproject/amp-toolbox/blob/master/ssr/lib/DomTransformer.js#L39-L44), not the internal transformers.